pinot-common/src/test/java/org/apache/pinot/common/utils/UtilsTest.java

```diff
@@ -24,48 +24,53 @@
 import org.testng.Assert;
 import org.testng.annotations.Test;
 
+import static org.testng.Assert.assertEquals;
```

Comment:
Is there any benefit on this except we can use `assertEquals` instead of `Assert.assertEquals`?

Reply:
No. I just prefer importing static Assert methods because IDE does so when creating test automatically.